docs/source/visualization/ade_mosaic.ipynb

```diff
@@ -0,0 +1,259 @@
+    "3. Combine the MosaicJson with other tiler visualization parameters to register a layer with your visualization tool.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Package library requirement needed on new workspace\n",
+    "#%conda install -c conda-forge cogeo-mosaic\n",
+    "!pip install cogeo-mosaic"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "import urllib\n",
+    "\n",
+    "from cogeo_mosaic.mosaic import MosaicJSON\n",
+    "from cogeo_mosaic.backends import MosaicBackend"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Build a list of files\n",
+    "\n",
+    "You can either make a list of file paths, or create a geojson layer with a column containing the file paths.\n",
+    "The paths need to be **s3** paths currently."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['s3://maap-ops-dataset/maap-users/alexdevseed/landsat8/viz/Landsat8_30542_comp_cog_2015-2020_dps.tif',\n",
+       " 's3://maap-ops-dataset/maap-users/alexdevseed/landsat8/viz/Landsat8_30543_comp_cog_2015-2020_dps.tif',\n",
+       " 's3://maap-ops-dataset/maap-users/alexdevseed/landsat8/viz/Landsat8_30822_comp_cog_2015-2020_dps.tif',\n",
+       " 's3://maap-ops-dataset/maap-users/alexdevseed/landsat8/viz/Landsat8_30823_comp_cog_2015-2020_dps.tif']"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Local Path to your COGs\n",
+    "dps_output = \"/projects/maap-users/alexdevseed/landsat8/viz\"\n",
+    "\n",
+    "# Search for files to include, use recursive if nested folders (common in DPS output)\n",
+    "files = glob.glob(os.path.join(dps_output, \"Landsat*_dps.tif\"), recursive=False)\n",
+    "\n",
+    "def local_to_s3(url):\n",
+    "    ''' A Function to convert local paths to s3 urls'''\n",
+    "    return url.replace(\"/projects/maap-users\", \"s3://maap-ops-dataset/maap-users\")\n",
+    "\n",
+    "tiles = [local_to_s3(file) for file in files]\n",
+    "print(tiles)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How to find the s3 path\n",
+    "\n",
+    "You might be wondering how to find the s3 path to begin with. Right now the easiest way is to right click on a file in the file explorer on the left panel, and Get Presigned S3 Url.\n",
+    "\n",
+    "It will look something like `https://maap-ops-dataset.s3.amazonaws.com/maap-users/alexdevseed/landsat8/viz/Landsat8_30542_comp_cog_2015-2020_dps.tif?AWSAccessKeyId...`\n",
+    "\n",
+    "The first part of the url is the bucket name: `maap-ops-dataset`\n",
```

Comment:
Appears there is missing period at end of sentence - https://github.com/MAAP-Project/maap-documentation/blob/feat/ADEmosaic/docs/source/visualization/ade_mosaic.ipynb